templates/udev_rule.erb

```diff
@@ -0,0 +1,4 @@
+# This file managed by Puppet - DO NOT EDIT
```

Comment:
Why not use EPP?

Reply:
Old habbits die hard.... I'll switch over to EPP and have a new version of this up soon.